modules/bpi/bpi.admin.inc

```diff
@@ -198,6 +206,20 @@ function bpi_admin_content_mapping_form($form, $form_state) {
     '#default_value' => variable_get('bpi_field_materials', ''),
   );
 
+  $form['bpi_syndication'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Syndication settings'),
+    '#prefix' => '<div id="bpi-syndication-settings-wrapper">',
+    '#suffix' => '</div>',
```

Comment:
The wrapper was added for consistency with #bpi-field-mapper-wrapper. I'll remove the wrapper.